.github/workflows/merge-to-master.yaml

```diff
@@ -6,8 +6,7 @@ on:
       - master
 
 env:
-  SDK_VERSION: "1.3.0"
-  OPM_VERSION: "1.15.2"
```

Comment:
Why remove `OPM_VERSION`?

Reply:
It is not used in GH actions any more

Reply:
See https://github.com/redhat-developer/service-binding-operator/pull/1094/files#diff-2b70abc98e31f0c8655218ca51b2babe88c6d006a9354d519694d74c106e9a9eL12 or https://github.com/redhat-developer/service-binding-operator/pull/1094/files#diff-4960229ff0042669817882805912433abf5b83741ac0d9a9f03ad70aefd2a925L10 for reference